website/screens/components/dialog/code/examples/withContent.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  DxcDialog,
+  DxcButton,
+  DxcTextInput,
+  DxcInset,
+  DxcRow,
+} from "@dxc-technology/halstack-react";
+import { useState } from "react";
+
+const code = `() => {
+  const [isDialogVisible, setDialogVisible] = useState(false);
+  const handleClick = () => {
+    setDialogVisible(!isDialogVisible);
+  };
+  return (
+    <DxcInset space="large">
+      <DxcRow justify="center">
+        <DxcButton label="Open Dialog" onClick={handleClick}></DxcButton>
+        {isDialogVisible && (
+          <DxcDialog onCloseClick={handleClick}>
+            <DxcRow>
+              <DxcTextInput
+                label="Name"
+                margin={{ bottom: "medium", right: "small" }}
+              />
+              <DxcTextInput label="Last name" margin={{ bottom: "medium" }} />
+            </DxcRow>
+            <DxcRow>
+              <DxcTextInput
+                label="Address"
+                margin={{ bottom: "medium" }}
+                size="fillParent"
+              />
+            </DxcRow>
+            <DxcRow justify="center">
+              <DxcButton label="Add client" onClick={handleClick} />
+              <DxcButton label="Cancel" onClick={handleClick} mode="text" />
+            </DxcRow>
+          </DxcDialog>
```

Comment:
I would not use DxcRow in order to have it not centered, as in the rest of components.
How do you see if instead of putting margins to text inputs we use an inset for each one?